packages/trader/src/Modules/SmartChart/Components/Markers/__tests__/accumulators-chart-elements.spec.tsx

```diff
@@ -5,19 +5,30 @@ import AccumulatorsChartElements from '../accumulators-chart-elements';
 jest.mock('App/Components/Elements/PositionsDrawer/helpers/positions-helper.js', () => ({
     filterByContractType: jest.fn(() => true),
 }));
-jest.mock('../accumulators-profit-loss-tooltip.jsx', () => () => <div>AccumulatorsProfitLossTooltip</div>);
-jest.mock('../current-spot-emphasizer.jsx', () => () => <div>CurrentSpotEmphasizer</div>);
+jest.mock('../accumulators-profit-loss-tooltip', () => jest.fn(() => <div>AccumulatorsProfitLossTooltip</div>));
+jest.mock('../current-spot-emphasizer.jsx', () => jest.fn(() => <div>CurrentSpotEmphasizer</div>));
 
 describe('AccumulatorsChartElements', () => {
     const mock_props = {
```

Comment:
any types for props?

Reply:
it's optional to add types for props because TS can figure out what kind of props the component expects when we import it.